nanoFramework.Device.Bluetooth/BluetoothEventListener.cs

```diff
@@ -23,7 +21,7 @@ internal class BluetoothEventListener : IEventProcessor, IEventListener, IDispos
         public BluetoothEventListener(INativeDevice nativeDevice = null)
```

Comment:
in that case of changing and creating a breaking change, then remove the default null as it does not make sense to pass it and generate an exception.

Reply:
Also add the exception in the intellisense description. We're now following this pattern and removing as much text as possible from the exception themselves to gain space on our lovely devices